### ISSUE TITLE
feat(accessibility): Resolve critical accessibility issues

### ISSUE THREAD

**Issue by scarletmerlin123** [OpenCTI-Platform/opencti#17312]
## Use case

Upon scanning the application for section 508 compliance multiple critical accessibility issues were found. 

- Missing focus indicators
- Incorrect use of Modal component without a focus trap, dialog role, or close button
- QR code images with blank alt text
- OTP input cells with no accessible label
- Clickable div elements used as interactive controls
- Insufficient contrast for helper text
- Insufficient contrast for the severity "medium" color in the light theme 

## Current workaround

None

## Proposed solution

Resolve critical accessibility issues to meet WCAG guidelines

## If the feature request is approved, would you be willing to submit a PR?

Yes
